FIFORhymer.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.vh.nursery;
+
+import edu.kis.vh.nursery.collections.IntLinkedList;
+
+/**
+ * @author Radek
+ * Klasa FIFORhymer dziedziczy po klasie DefaultCountingOutRhymer
+ */
+public class FIFORhymer extends DefaultCountingOutRhymer {
+
+	
+	public FIFORhymer() {
+		super();
+		this.temp = new IntLinkedList();
+		// TODO Auto-generated constructor stub
+	}
+
+
+	public FIFORhymer(IntLinkedList intLinkedList) {
+		super(intLinkedList);
+		// TODO Auto-generated constructor stub

```

Comment:
missing: this.temp = new IntLinkedList(); (probably throws exception during work)